garage-door-GDOv2-Q.yaml

```diff
@@ -193,6 +193,6 @@ web_server:
 esphome:
   platformio_options:
     lib_deps:
-      - https://github.com/konnected-io/gdolib
+      - https://github.com/konnected-io/gdolib#command-delay-rx-fix
```

Comment:
if godlib is merged before this code, or gdolib is refreshed/updated later, will things break?

Reply:
gdolib changes should be merged first, however this will no break unless that branch is deleted.